tests/helpers.test.js

```diff
@@ -0,0 +1,511 @@
+const _ = require("lodash");
+const consts = require("../consts.json");
+const helpers = require("../helpers");
+const rewire = require("rewire");
+
+describe("removeCredentials", () => {
+  test("properly removes credentials from params based on DEFAULT_CREDENTIAL_LABELS if no labels provided", () => {
+    const params = {
+      [consts.DEFAULT_CREDENTIAL_LABELS.ACCESS_KEY]: "test access",
+      [consts.DEFAULT_CREDENTIAL_LABELS.SECRET_KEY]: "test secret",
+      [consts.DEFAULT_CREDENTIAL_LABELS.REGION]: "test region",
+      otherParam: "test param",
+    };
+
+    const returnedParams = helpers.removeCredentials(params);
+    expect(_.keys(returnedParams)).toStrictEqual(["otherParam"]);
+  });
+
+  test("properly removes credentials from params based on provided labels", () => {
+    const labels = {
+      ACCESS_KEY: "access",
```

Comment:
shouldn't `consts` be used here?

Reply:
I mean this test is here to see if user provided labels would work, so I figured that not using `consts` actually better simulates this case.